src/cript/api/exceptions.py

```diff
@@ -0,0 +1,61 @@
+class CRIPTConnectionError(Exception):
```

Comment:
you should make a base 'CRIPTException' class and have these error class inherit from that. This makes it easier for future work to catch all exceptions create by CRIPT SDK